OWNERS_ALIASES

```diff
@@ -62,15 +62,10 @@ aliases:
     - soltysh
     - sttts
   sig-cluster-lifecycle: #GH: sig-cluster-lifecycle-pr-reviews
-    - jbeda
     - timothysc
-    - lukemarsden
-    - pipejakob
-    - dmmcquay
-    - mattmoyer
     - luxas
     - roberthbailey
-    - medinatiger
+    - bradamant3
```

Comment:
Fine to remove a reviewer, but please don't add me. We use this list to specify tech reviewers, and I don't think I should be in that list ;)